lib/PAUSE/dist.pm

```diff
@@ -410,214 +468,71 @@ sub mail_summary {
 
   my $status_over_all;
 
-  if (my $err = $self->{REASON_TO_SKIP}) {
-    push @m, $tf->format( PAUSE::mldistwatch::Constants::heading($err) ),
-             qq{\n\n};
-    $status_over_all = "Failed";
-  }
-
-  # NO_DISTNAME_PERMISSION must not hide other problem messages, so
-  # we fix up any "OK" status records to reflect the permission
-  # problem and let the rest of the report run as usual
-  if ($self->{NO_DISTNAME_PERMISSION}) {
-    my $pkg = $self->_package_governing_permission;
-    push @m, $tf->format(qq[This distribution name will only be indexed
-      when uploaded by users with permission for the package $pkg.
-      Either someone else has ownership over that package name, or
-      this is a brand new distribution and that package name was neither
-      listed in the 'provides' field in the META file nor found
-      inside the distribution's modules.  Therefore, no modules
-      will be indexed.]);
-    push @m, qq{\nFurther details on the indexing attempt follow.\n\n};
-    $status_over_all = "Failed";
-
-    my $inxst = $self->{INDEX_STATUS};
-    if ($inxst && ref $inxst && %$inxst) {
-      unless ($inxst->{$pkg}) {
-        # Perhaps they forgot a pm file matching the dist name
-        my($inxpkg_eg) = sort keys %$inxst;
-        $inxpkg_eg =~ s/::/-/g;
-        $inxpkg_eg =~ s/$/-.../;
-        push @m, $tf->format(qq{\n\nYou appear to be missing a .pm file
-          containing a package matching the dist name ($pkg). Adding this
-          may solve your issue. Or maybe it is the other way round and a
-          different distribution name could be chosen to reflect an
-          actually included package name (eg. $inxpkg_eg).\n});
-    }
-
-      for my $p ( keys %$inxst ) {
-          next unless
-            $inxst->{$p}{status} == PAUSE::mldistwatch::Constants::OK;
-          $inxst->{$p}{status} = PAUSE::mldistwatch::Constants::EDISTNAMEPERM;
-          $inxst->{$p}{verb_status} =
-            "Not indexed; $author not authorized for this distribution name";
```

Comment:
It seems like a user won't see a not authorized error anymore?

Reply:
They'll see it in the whole-dist context, and there is no longer a per-module error in the "no distname permissions" case.